lib/AuditoManager.dart

```diff
@@ -52,53 +25,37 @@ class AudioManager {
         isPlaying = false;
       }
     });
-    print("Playing audio: $audioFile");
+    debugPrint("Playing audio: $audioFile");
   }
 
   Future<void> pauseAudio() async {
     await _audioPlayer.pause();
     isPlaying = false;
-    print("Audio paused.");
+    debugPrint("Audio paused.");
   }
 
   Future<void> resumeAudio() async {
     await _audioPlayer.resume();
     isPlaying = true;
-    print("Audio resumed.");
+    debugPrint("Audio resumed.");
   }
 
   Future<void> stopAudio() async {
     await _audioPlayer.stop();
     isPlaying = false;
-    print("Audio stopped.");
+    debugPrint("Audio stopped.");
   }
 
   Future<void> adjustVolume(double level) async {
     volume = level.clamp(0.0, 1.0);
     await _audioPlayer.setVolume(volume);
-    await _database?.update('preferences', {'volume': volume}, where: 'id = ?', whereArgs: [1]);
-    print("Volume set to ${(volume * 100).toStringAsFixed(0)}% and saved in preferences.");
-  }
-
-  Future<void> addMindfulnessCue(int sessionId, int timeSec, String message) async {
-    await _database?.insert(
-      'cues',
-      {'session_id': sessionId, 'time_sec': timeSec, 'message': message},
-    );
-    print("Cue added at $timeSec seconds: $message");
-  }
-
-  Future<int> fetchAudioDuration(String audioFile) async {
-    // Assuming audio duration retrieval via a library or pre-known metadata
-    return await _audioPlayer.getDuration() ?? 0; // Returns duration in milliseconds
+    debugPrint("Volume set to ${(volume * 100).toStringAsFixed(0)}%");
   }
 
-  Future<List<Map<String, dynamic>>> getMindfulnessCues(int sessionId) async {
-    final List<Map<String, dynamic>> cues = await _database?.query(
-          'cues',
-          where: 'session_id = ?',
-          whereArgs: [sessionId],
-        ) ?? [];
-    return cues;
+  // Dummy function to demonstrate preference fetching
+  Future<void> loadVolumeFromPreferences() async {
+    // Example: Assuming volume is fetched from storage.dart preferences
+    // volume = await Storage.getVolumePreference();
+    debugPrint("Volume loaded from preferences: $volume");
```

Comment:
If the function is only going to run a single print statement, then don't have it at all.

If you want to have example code, put it in a comment or a readme